frontend/src/components/reports/PendingReportsTab.vue

```diff
@@ -5,73 +5,75 @@
         <h2>Report Details</h2>
       </v-col>
       <v-col cols="12" md="7">
-        <FacilityFilter :loading="loading" :defaultShowInput="true" justify="end" @facility-filter-changed="facilityFilterChanged" />
+        <FacilityFilter v-if="!isEmpty(pendingReports)" :loading="loading" :defaultShowInput="true" justify="end" @facility-filter-changed="facilityFilterChanged" />
       </v-col>
     </v-row>
 
-    <v-skeleton-loader :loading="processing" type="table-tbody">
-      <v-data-table :headers="headers" :items="filteredPendingReports" item-key="surveyResponseReferenceNumber" density="compact">
-        <template #[`item.alertType`]="{ item }">
-          <v-icon v-if="item.alertType === 'Due'" color="warning">mdi-alert</v-icon>
-          <v-icon v-else-if="item.alertType === 'Overdue'" color="error">mdi-alert-circle</v-icon>
-          <span class="pl-1">{{ item.alertType }}</span>
+    <v-skeleton-loader :loading="loading" type="table-tbody">
+      <AppAlertBanner v-if="isEmpty(pendingReports)" type="info">
+        <div>You are up to date with your monthly reports.</div>
+      </AppAlertBanner>
+      <v-data-table v-else :headers="headers" :items="filteredPendingReports" item-key="surveyTemplateId" density="compact" class="soft-outline">
+        <template #[`item.alert`]="{ item }">
+          <span v-if="isOverdue(item)">
+            <v-icon color="error">mdi-alert-circle</v-icon>
+            Overdue
+          </span>
+          <span v-else>
+            <v-icon color="warning">mdi-alert</v-icon>
+            Due
+          </span>
         </template>
-        <template #[`item.status`]="{ item }">
-          <span :class="{ 'status-gray': item.status === SURVEY_RESPONSE_STATUSES.DRAFT }" class="px-2 py-1">{{ item.status }}</span>
+        <template #[`item.title`]="{ item }">
+          <span>{{ getReportTitle(item) }}</span>
+        </template>
+        <template #[`item.status`]>
+          <span class="status-gray">Draft</span>
         </template>
         <template #[`item.actions`]="{ item }">
-          <v-btn v-if="showOpen(item)" icon="mdi-folder-open-outline" variant="text" @click="openSurveyResponse(item)" />
-          <v-btn v-if="showTrash(item)" icon="mdi-trash-can-outline" variant="text" @click="toggleCancelDialog" />
+          <v-btn variant="text" @click="openSurveyResponse(item)">
```

Comment:
I think there's a related story to update this to an Open button. If we do that we should consider either using AppButton or a new AppRowButton which we use for row button styling. Let's chat.

Reply:
Yes, I've let QA team know that those buttons will be updated in this ticket OFMCC-4146. I've tried use AppButton but the styling looks off, so I decided to update those icons to text in 4146.